Expensify/App
# Feature: Transfer balance

### Details


### Fixed Issues

$ #3922

### Tests | QA Steps
1. Open Payments page.
2. Click TransferBalance Link.
3. Confirm that the transfer button is disabled if the following conditions are true.
  i. Transfer Amount is less than $ 0.30.
  ii. Which Account is not selected.
4. Click instant Mode. 
5. It should take you to the new page where you will select an account. This page should show you Debit cards only and Add Debit Card Button.
6. Clicking  `+ Add Debit Card` Button opens a new page to add Debit Card.
7. From the Select Account page, click on the any debit card. 
8. You should be taken to the old page where you can see the selected debit card as `Which account?`
9. If your balance is > Fee then `Transfer` Button should be active and it should show the transfer amount - fee. 
4. Click 1-3 Business Days Mode. 
5. It should take you to the new page where you will select an account. This page should show you Bank Accounts only and Add bank account Button.
6. Clicking  `+ Add bank Account` Button opens a new page to add bank Account.
7. From the Select Account page, click on any bank account. 
8. You should be taken to the old page where you can see the selected bank account card as `Which account?`
9. If your balance is > Fee then `Transfer` Button should be active and it should show the transfer amount - fee. 

10. If your balance is > Fee and which account is set.
 i. Click Transfer button.
 ii. Now you should be taken to the Payments Page and a confirm modal should be shown.
 
### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/24370807/145723085-ddec27cd-2e8b-4a7c-80e4-76bf04c5a923.mp4

#### Mobile Web


#### Desktop


#### iOS


#### Android
